tests/forms/test_validators.py

```diff
@@ -52,3 +55,55 @@ def test_email_validator_does_not_raise_on_a_valid_email_address(form, field, em
     field.data = email_address
 
     validator(form, field)
+
+
+class TestGreaterThan:
```

Comment:
Should one set of parameters be a date format, since we use this on those questions?

I'm not sure if we're comparing strings or numbers (or date objects?) in that case.

Reply:
Sure, it shouldn't in theory make a difference, but it definitely makes sense to test it!